stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+
+/**
+ * Interface that callers need to implement in order to pretty print the payload received by Stetho
+ */
+public interface AsyncPrettyPrinter {
+  /**
+   * Prints the prettified version of payload to output
```

Comment:
Explain that this method can block for a period of time, but warn implementors that the timeout may be arbitrarily imposed by stetho.